lib/cluster.js

```diff
@@ -119,7 +119,12 @@ Cluster.prototype.fetchStats = function fetchStats(callback) {
     var allStats = [];
     var fetchQueue = async.queue(mapMember, 5);
 
+    var connectedToMember = false;
+
     fetchQueue.drain = function () {
+        if (!connectedToMember) {
+            callback(new Error('Failed to connect to any ringpop members'));
```

Comment:
This is going to result in `callback()` being called twice since you're not returning in the if-block.

Should you use onComplete(new Error(...)) instead?